SparseGrids/tsgGridLocalPolynomial.hpp

```diff
@@ -191,15 +198,32 @@ class GridLocalPolynomial : public BaseCanonicalGrid{
 
         for(const auto &r : roots){
             bool isSupported;
-            double basis_value = evalBasisSupported(work.getIndex(r), x, isSupported);
+            double basis_value;
```

Comment:
Uninitialized `basis_value`. Initialize inline with `() ? : ` expression and set to zero for modes 3 and 4.

Reply:
Changed.